src/openassetio-core/hostApi/ManagerFactory.cpp

```diff
@@ -56,8 +56,15 @@ ManagerFactory::ManagerDetails ManagerFactory::availableManagers() const {
 }
 
 ManagerPtr ManagerFactory::createManager(const Identifier& identifier) const {
-  return Manager::make(managerInterfaceFactory_->instantiate(identifier),
-                       managerApi::HostSession::make(managerApi::Host::make(hostInterface_)));
+  return createManagerForInterface(hostInterface_, managerInterfaceFactory_, identifier);
+}
+
+ManagerPtr ManagerFactory::createManagerForInterface(
+    const HostInterfacePtr& hostInterface,
+    const ManagerInterfaceFactoryInterfacePtr& managerInterfaceFactory,
+    const ManagerFactory::Identifier& identifier) {
```

Comment:
[fix] In our "order arguments as per the name" convention, would it make sense to order the arguments as:

`createManagerForInterface(identifier, hostInterface, factory, logger)`

It also reenforce that this is an alternate version of `createManager` where the other two arguments are held by the factory, and perhaps reduces that ambiguity around the two interfaces?

Reply:
Good point, updated in a961620b